OpenRA.Mods.D2k/Traits/AutoCarryall.cs

```diff
@@ -30,9 +34,12 @@ public class AutoCarryall : INotifyBecomingIdle, INotifyKilled, ISync, IRender
 	{
 		readonly Actor self;
 		readonly WRange carryHeight;
+		readonly AutoCarryallInfo info;
 
 		// The actor we are currently carrying.
-		[Sync] Actor carrying;
+		[Sync] public Actor Carrying { get; internal set; }
+
+		public bool HasCarryableAttached { get; internal set; }
```

Comment:
No, because Carrying gets set when the carryable gets reserved - its not necessarily attached.